Sources/Sentry/SentrySession.m

```diff
@@ -186,6 +186,28 @@ - (void)incrementErrors
     }
 }
 
+- (id)copyWithZone:(nullable NSZone *)zone
+{
+    SentrySession *copy = [[[self class] allocWithZone:zone] init];
+
+    if (copy != nil) {
+        copy->_sessionId = _sessionId;
+        copy->_started = _started;
+        copy->_status = _status;
+        copy->_errors = _errors;
+        copy->_sequence = _sequence;
+        copy->_distinctId = _distinctId;
+        copy->_timestamp = _timestamp;
+        copy->_duration = _duration;
+        copy->_releaseName = _releaseName;
+        copy.environment = self.environment;
+        copy.user = self.user;
```

Comment:
This would need to be a clone too, right?

Actually I'm even surprised we have a reference to the user in the session, I thought we removed this.

Reply:
Ops, yes this should be copied too. I don't know if we should have a reference here or not. In [Python](https://github.com/getsentry/sentry-python/blob/9abf26ba003a7cc101e246f1bb4413d4f72fab40/sentry_sdk/sessions.py#L136) we do, but on [Android](https://github.com/getsentry/sentry-android/blob/168a8460ffe2e86b81e549a5e67c587034f2ef50/sentry-core/src/main/java/io/sentry/core/Session.java#L106), for example, we just keep the IP address. 
Do you know which one is the right way? If we want to remove the user, I would do this in another PR.

Reply:
I just found out that the property [SentrySession.user has the copy attribute](https://github.com/getsentry/sentry-cocoa/blob/6.0.0/Sources/Sentry/include/SentrySession.h#L41). Therefore we don't need to call copy in here, because it is already copied when you assign the user to the session.

Reply:
I don't believe we use `user` at all though, this was here from v1, before installation Id. Not sure..
I'd say it's OK to always copy values to avoid surprises if things changed.